src/typescript/providers/src/wix/config.interface.ts

```diff
@@ -0,0 +1,47 @@
+import type {Variable} from 'ddns-base';
+
+import type {ProviderBaseConfig} from '../provider.interface';
+
+export interface WixProviderConfig extends ProviderBaseConfig {
+    /**
+     * The account ID linked to the API key used when updating DNS records on Wix.
+     * @example {"from": "Env", "name": "WIX_ACCOUNT_ID"}
+     */
+    accountId: Variable;
+
+    /**
+     * The API key used when updating DNS records on Wix.
+     * Needs All account permissions > Manage Domains.
+     * Needs to be created using the account of the owner of the site to which the domain is linked.
```

Comment:
This owner/account/link/domain is not very clear. Rewrite and explain the limitation of co-owners.